.then(data=>console.log('promise resolved',data)
)
.catch(error => console.log(error)
)